sematic/ui/src/appContext.ts

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export const AppContext = React.createContext<{
+    authenticatedEnabled: boolean;
+    googleAuthClientId: string | undefined;
```

Comment:
Can we make this more abstract? In the future, we will have potentially multiple auth mechanisms (e.g. Google, GitHub, Slack, etc.). Shouldn't this be an `authProviderDetails` where keys would be `google`, `github`, `slack`, and values would be a dict of relevant details (e.g. client IDs).

Reply:
Sure, how does it look now?